jira7/jira_endpoint.bal

```diff
@@ -30,45 +30,44 @@ public type Client client object {
 
     # Returns an array of all projects summaries which are visible for the currently logged in user who has
     # `BROWSE`, `ADMINISTER` or `PROJECT_ADMIN` project permission.
-    # + return - Array of `ProjectSummary` objects or `JiraConnectorError` record
-    public remote function getAllProjectSummaries() returns ProjectSummary[]|JiraConnectorError;
+    # + return - Returns Array of `ProjectSummary` records when successful, else returns an error
```

Comment:
We don't need to add the "Returns " part IMO, since it is clear we are documenting the return value.

```suggestion
    # + return - An array of `ProjectSummary` records if successful, else returns an error
```

Shall we change "when" to "if" for all the new changes and remove the "Returns ...." additions here?